core/src/test/java/com/graphhopper/reader/dem/SRTMGL1ProviderTest.java

```diff
@@ -0,0 +1,70 @@
+/*
+ *  Licensed to GraphHopper GmbH under one or more contributor
+ *  license agreements. See the NOTICE file distributed with this work for 
+ *  additional information regarding copyright ownership.
+ * 
+ *  GraphHopper GmbH licenses this file to you under the Apache License, 
+ *  Version 2.0 (the "License"); you may not use this file except in 
+ *  compliance with the License. You may obtain a copy of the License at
+ * 
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package com.graphhopper.reader.dem;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * @author Robin Boldt
+ */
+public class SRTMGL1ProviderTest {
```

Comment:
I wonder: when we added a test isn't there also a file required under core/files/xy.hgt? Otherwise fresh tests won't run without internet or could be slow (?)

Reply:
All elevation provider (except the SRTMProvider) tests don't actually return the height of a coordinate, they only return the name of the file they are going to download. I haven't seen that there are files for the srtm provider. Should we add files for the SRTMGL1 provider as well? (note, the files are 25mb large and per default not zipped). CGIAR files are also around 30mb. GMTED are >200mb. Not sure if it's easy to create small dummy files?

This is actually something that bothers me as well. I was thinking about adding a test, but to put @Ignore at the test, so we can enable the test if we change something about the elevation provider, but don't use it most of the time. WDYT?

Reply:
There are files at core/files/*hgt.zip

> Should we add files for the SRTMGL1 provider as well?

Ok, probably not :)

The idea with the `@Ignore` seems interesting but I think there is something in combination with maven for executing only a specific set of tests

> BTW: Our API/client tests probably fail if there is no internet as well (haven't tested that yet I think)?

Yes, we should include an option to exclude it or just run it on travis?